src/index.html

```diff
@@ -11,7 +11,179 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="box-container">
+        <legend>Personal information</legend>
+        <label
+        class="form-label"
+        for="surname">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            id="surname"
+            minlength="2"
+            maxlength="30"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="name">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            id="name"
+            minlength="2"
+            maxlength="30"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="age">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            id="age"
+            step="1"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="date-of-birth">
+          Full date of birth:
+          <input
+            type="date"
+            name="data-of-birth"
```

Comment:
tiny but important details. 2 in 1:
- `date-of-birth` vs `data-of-birth`
- `label for` without `input id`
```suggestion
        <label class="form-label" for="date-of-birth">
          Full date of birth:
          <input
            type="date"
            name="date-of-birth"
            id="date-of-birth"
```